docs/integrations_api.yml

```diff
@@ -0,0 +1,134 @@
+swagger: '2.0'
+info:
+  version: '0.1'
+  title: User administration and authentication
+  description: |
+    An API for user administration and user authentication handling. Intended for use by the web GUI.
+
+basePath: '/api/integrations/0.1/users'
```

Comment:
could be useradm just like a repo, burt don't have any particular solution, brought this up as I just noticed this `.../user/user/..`